File: src/main/java/com/larryhsiao/clotho/color/BlueInt.java
package com.larryhsiao.clotho.color;

import com.larryhsiao.clotho.Source;

/**
 * Extract blue int in color int
 */
public class BlueInt implements Source<Long> {
    private final long color;

    public BlueInt(long color) {this.color = color;}

    @Override
    public Long value() {
        return color & 0xff;
    }
}

File: src/main/java/com/larryhsiao/clotho/color/BlueInt.kt
package com.larryhsiao.clotho.color

import com.larryhsiao.clotho.Source

/**
 * Extract blue int in color int
 */
class BlueInt(private val color: Long) : Source<Long> {
    override fun value(): Long {
        return color and 0xff
    }
}